src/index.html

```diff
@@ -35,118 +40,121 @@ <h2>Data</h2>
       </div>
       <div class="nav-links">
         <!-- @todo confirm feedback URL -->
-        <a class="thumbs-up" href="mailto:lightbeam-feedback@mozilla.org">Give Us Feedback</a>
+        <!-- <a class="thumbs-up" href="mailto:lightbeam-feedback@mozilla.org">Give Us Feedback</a> -->
```

Comment:
Rather than commenting this, how about a link back to the issues page on the project?

Reply:
Makes a lot of sense, will do.